### DISCUSSION TITLE
Education and no Copilot

### DISCUSSION THREAD

**Discussion by nab1g24** [community/community#145426] [Copilot Conversations]
### Select Topic Area

General

### Body

I have just been approved for GitHub Copilot, and this meant I would get free access to Github Copilot. In reality, I cannot see how to use it, and the only suggestion is the free trial and the intro to Copilot; the latter requires access to Copilot to complete.

**Comment by davevad93** [comment]
Hi @nab1g24 ,

I guess you're talking about GitHub Education benefits, make sure the status label is purple and states "`Congrats! Your request for @nab1g24 was approved and your benefits are now available.`", if so, try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), by doing that you should be able to redeem your offer, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Reply by nab1g24** [reply]
Thank you. I have got it working now based on your advice. And I also appreciate that my original message might not have been all that clear as I wasn't sure how to describe my situation.

**Reply by davevad93** [reply]
You're welcome @nab1g24 , glad to help! 👍
